paket.dependencies

```diff
@@ -4,9 +4,10 @@ nuget FAKE
 nuget Paket.Core
 nuget FSharp.Formatting
 nuget VSSDK.GraphModel.12 copy_local: false
-nuget VSSDK.Text.12 copy_local: false
```

Comment:
Looks like 'paket simplify' might have hit that one.  I'll add it back in when I get home.